src/main/scala/com/cognite/sdk/scala/v1/containers/PropertyDefaultValue.scala

```diff
@@ -0,0 +1,34 @@
+// Copyright 2020 Cognite AS
+// SPDX-License-Identifier: Apache-2.0
+
+package com.cognite.sdk.scala.v1.containers
+
+import io.circe._
+
+import java.util.Locale
+
+sealed abstract class PropertyDefaultValue extends Product with Serializable
+
+object PropertyDefaultValue {
```

Comment:
I believe this is incorrect. This must be decoded to arbitrary values rather than enum. We could make use of `DataModelProperty[_]` from earlier but not sure if it would be easier to do it from start. I am a little confused myself... 😓